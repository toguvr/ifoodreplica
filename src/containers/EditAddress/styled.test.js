import React from 'react'
import renderer from 'react-test-renderer';
import { ContainerEditAddressPage, InputAddress, InputNumber, InputHouse, InputNeighborhood, InputCity, InputState } 
from './styled';

describe("ContainerEditAddressPage", () => {
    it("Should match Snapshot", () => { 
        const tree = renderer
        .create(<ContainerEditAddressPage />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("InputAddress", () => {
    it("Should match Snapshot", () => { 
        const tree = renderer
        .create(<InputAddress />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})


describe("InputNumber", () => {
    it("Should match Snapshot", () => { 
        const tree = renderer
        .create(<InputNumber />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})


describe("InputHouse", () => {
    it("Should match Snapshot", () => { 
        const tree = renderer
        .create(<InputHouse />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("InputNeighborhood", () => {
    it("Should match Snapshot", () => { 
        const tree = renderer
        .create(<InputNeighborhood />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("InputCity", () => {
    it("Should match Snapshot", () => { 
        const tree = renderer
        .create(<InputCity />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("InputState", () => {
    it("Should match Snapshot", () => { 
        const tree = renderer
        .create(<InputState />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})