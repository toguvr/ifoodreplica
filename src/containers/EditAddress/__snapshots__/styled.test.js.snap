// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContainerEditAddressPage Should match Snapshot 1`] = `
<div
  className="sc-bdVaJa NWYyK"
/>
`;

exports[`InputAddress Should match Snapshot 1`] = `
<div
  className="MuiFormControl-root-1 sc-bwzfXH hGBRVj"
>
  <div
    className="MuiInputBase-root-18 MuiInput-root-5 MuiInput-underline-9 MuiInputBase-formControl-19 MuiInput-formControl-6"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      className="MuiInputBase-input-28 MuiInput-input-13"
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
    />
  </div>
</div>
`;

exports[`InputCity Should match Snapshot 1`] = `
<div
  className="MuiFormControl-root-1 sc-EHOje fBcYxG"
>
  <div
    className="MuiInputBase-root-18 MuiInput-root-5 MuiInput-underline-9 MuiInputBase-formControl-19 MuiInput-formControl-6"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      className="MuiInputBase-input-28 MuiInput-input-13"
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
    />
  </div>
</div>
`;

exports[`InputHouse Should match Snapshot 1`] = `
<div
  className="MuiFormControl-root-1 sc-bxivhb iomlTu"
>
  <div
    className="MuiInputBase-root-18 MuiInput-root-5 MuiInput-underline-9 MuiInputBase-formControl-19 MuiInput-formControl-6"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      className="MuiInputBase-input-28 MuiInput-input-13"
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
    />
  </div>
</div>
`;

exports[`InputNeighborhood Should match Snapshot 1`] = `
<div
  className="MuiFormControl-root-1 sc-ifAKCX bQwSPf"
>
  <div
    className="MuiInputBase-root-18 MuiInput-root-5 MuiInput-underline-9 MuiInputBase-formControl-19 MuiInput-formControl-6"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      className="MuiInputBase-input-28 MuiInput-input-13"
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
    />
  </div>
</div>
`;

exports[`InputNumber Should match Snapshot 1`] = `
<div
  className="MuiFormControl-root-1 sc-htpNat hYLqpk"
>
  <div
    className="MuiInputBase-root-18 MuiInput-root-5 MuiInput-underline-9 MuiInputBase-formControl-19 MuiInput-formControl-6"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      className="MuiInputBase-input-28 MuiInput-input-13"
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
    />
  </div>
</div>
`;

exports[`InputNumber Should match Snapshot 2`] = `
<div
  className="MuiFormControl-root-1 sc-htpNat hYLqpk"
>
  <div
    className="MuiInputBase-root-18 MuiInput-root-5 MuiInput-underline-9 MuiInputBase-formControl-19 MuiInput-formControl-6"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      className="MuiInputBase-input-28 MuiInput-input-13"
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
    />
  </div>
</div>
`;

exports[`InputState Should match Snapshot 1`] = `
<div
  className="MuiFormControl-root-1 sc-bZQynM kJaXIK"
>
  <div
    className="MuiInputBase-root-18 MuiInput-root-5 MuiInput-underline-9 MuiInputBase-formControl-19 MuiInput-formControl-6"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      className="MuiInputBase-input-28 MuiInput-input-13"
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
    />
  </div>
</div>
`;
